scripts/travis_script.sh

```diff
@@ -142,6 +142,12 @@ if [[ $GROUP == usage ]]; then
     jupyter lab workspaces export newspace > newspace.json
     rm workspace.json newspace.json
 
+    # Test pinned alias packages
+    jupyter labextension install --no-build \
+        --pin-version-as test-1,test-2 \
+        jupyterlab-test-extension@1.0 \
+        jupyterlab-test-extension@2.0
+    jupyter labextension uninstall test-1 test-2
```

Comment:
I think we can have silent failures here ("skipping already installed jupyterlab-test-extension"). Maybe it would be better to add a test to `jupyterlab/tests/test_jupyterlab`. Then we could also check that they are actually installed, and local tests will pick it up.

Reply:
I am a little confused, when should this silently fail? Currently, these tests shouldn't fail, since we add the `--pin-version-as`. Are you suggesting we  test if fails without this flag?

Reply:
I think moving them to the `jupyterlab/tests/test_jupyterlab` is a good idea, I will work on that.

Reply:
I have moved the tests over to bash.